docs/api/markdown.md

```diff
@@ -6,6 +6,15 @@ elements.
 
 ::: marimo.md
 
+## Loading LaTeX macros
+
+You can load LaTeX macros using `mo.latex(filename=...)`.
+
+::: marimo.latex
+
+!!! warning "Side effects"
```

Comment:
Maybe a runtime error if md case been called before latex- or is that too mean?

Reply:
probably too mean. i think it is ok